### PR TITLE
Add Ticket Availability and Logo entities

### PR DESCRIPTION
**Pull request by jmurphy45**
Add additional functionality to the wrapper for the most recent api.

### CODE REVIEW

**Comment by jmurphy45** [comment]
Fixed Address bug on needs to be address_1

**Comment by bigdevwhale** [comment]
Hi, @jmurphy45. I'll check up your pull request today or tomorrow.

File: src/Factories/Entity/TicketAvailability.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * Date: 2/13/20
+ * Time: 10:27 PM
+ */
+
+namespace Marat555\Eventbrite\Factories\Entity;
+
+
+class TicketAvailability extends AbstractEntity
+{
+//{"has_available_tickets":true,"minimum_ticket_price":{"currency":"USD","value":2000,"major_value":"20.00","display":"20.00 USD"},"maximum_ticket_price":{"currency":"USD","value":2000,"major_value":"20.00","display":"20.00 USD"},"is_sold_out":false,"start_sales_date":{"timezone":"America\/Chicago","local":"2020-01-10T10:00:00","utc":"2020-01-10T16:00:00Z"},"waitlist_available":false}
```
Comment (bigdevwhale) [inline src/Factories/Entity/TicketAvailability.php]: pls remove comments

File: src/Factories/Entity/TicketAvailability.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: josephmurphy
+ * Date: 2/13/20
+ * Time: 10:27 PM
+ */
```
Comment (bigdevwhale) [inline src/Factories/Entity/TicketAvailability.php]: pls remove autogenerated comments

File: src/Factories/Entity/Logo.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: josephmurphy
+ * Date: 2/13/20
+ * Time: 10:43 PM
+ */
```
Comment (bigdevwhale) [inline src/Factories/Entity/Logo.php]: pls remove autogenerated comments

File: src/Factories/Entity/TicketAvailability.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+class TicketAvailability extends AbstractEntity
+{
+//{"has_available_tickets":true,"minimum_ticket_price":{"currency":"USD","value":2000,"major_value":"20.00","display":"20.00 USD"},"maximum_ticket_price":{"currency":"USD","value":2000,"major_value":"20.00","display":"20.00 USD"},"is_sold_out":false,"start_sales_date":{"timezone":"America\/Chicago","local":"2020-01-10T10:00:00","utc":"2020-01-10T16:00:00Z"},"waitlist_available":false}
+
+    public $hasAvailableTickets;
+    public $waitlistAvailable;
+    public $isSoldOut;
+    public $minimumTicketPrice;
+    public $maxiumTicketPrice;
+    public $startSalesDate;
```
Comment (bigdevwhale) [inline src/Factories/Entity/TicketAvailability.php:48]: fix code style pls and add comments
Example:
`class CropMask extends AbstractEntity
{

    /**
     * The upload_token from the GET portion of the upload
     *
     * @var string
     */
    public $uploadToken;

    /**
     * Coordinate for top-left corner of crop mask
     *
     * @var TopLeft
     */
    public $topLeft;

    /**
     * Crop mask width
     *
     * @var number
     */
    public $width;

    /**
     * Crop mask height
     *
     * @var number
     */
    public $height;
}`

**Review by bigdevwhale — CHANGES_REQUESTED** [review]

File: src/Factories/Client.php [hunk 5]
```diff
@@ -4,6 +4,7 @@
 
 use GuzzleHttp\HandlerStack;
 use GuzzleHttp\Client as HttpClient;
+use Illuminate\Support\Facades\Log;
```
Comment (bigdevwhale) [inline src/Factories/Client.php]: pls delete this namespace because Log class is not needed here

File: src/Factories/Api/AbstractApi.php [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 namespace Marat555\Eventbrite\Factories\Api;
 
+use Illuminate\Support\Facades\Log;
```
Comment (bigdevwhale) [inline src/Factories/Api/AbstractApi.php]: pls delete this namespace because Log class is not needed here

File: src/Eventbrite.php [hunk 7]
```diff
@@ -2,6 +2,7 @@
 
 namespace Marat555\Eventbrite;
 
+use Illuminate\Support\Facades\Log;
```
Comment (bigdevwhale) [inline src/Eventbrite.php]: pls delete this namespace because Log class is not needed here

File: src/Factories/Entity/AbstractEntity.php [hunk 8]
```diff
@@ -54,6 +56,10 @@ public function build(array $parameters)
      */
     public function setProperty($property, $value)
     {
+//        Log::info($property);
```
Comment (bigdevwhale) [inline src/Factories/Entity/AbstractEntity.php]: pls delete debug code

File: src/Factories/Entity/AbstractEntity.php [hunk 9]
```diff
@@ -2,6 +2,8 @@
 
 namespace Marat555\Eventbrite\Factories\Entity;
 
+use Illuminate\Support\Facades\Log;
```
Comment (bigdevwhale) [inline src/Factories/Entity/AbstractEntity.php]: pls delete this namespace because Log class is not needed here

**Review by bigdevwhale — CHANGES_REQUESTED** [review]

**Pull request merged**